lib/Github/Api/Repo.php

```diff
@@ -60,6 +60,25 @@ public function all($id = null)
     }
 
     /**
+       * List all repositories owned by the user.
+       *
+       * @link https://developer.github.com/v3/repos/#list-your-repositories
+       *
+       * @param int|null $id The integer ID of the last Repository that you’ve seen.
```

Comment:
The order of the phpdoc parameters is wrong, the $id param doc should be moved to the last position